File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/examples/simple/DualIdentityMain.java
package net.fwitz.math.main.binary.dual.functions.examples.simple;

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualIdentityMain {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("z")
                .computeFn(z -> z)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/examples/simple/DualIdentityMain.kt
package net.fwitz.math.main.binary.dual.functions.examples.simple

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualIdentityMain {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("z")
        .computeFn { z -> z }
        .render()
}